benchmark.py

```diff
@@ -39,28 +41,41 @@ def RunScenario(scenario_config):
   if scenario_config.background_script is not None:
     subprocess.call(["osascript", f'./driver_scripts/{background_script}.scpt'])
 
-  subprocess.call(["osascript", f'./driver_scripts/{scenario_config.driver_script}.scpt'])
+  driver_script_args = ["osascript", f'./driver_scripts/{scenario_config.driver_script}.scpt']
+
+  process = subprocess.Popen(driver_script_args)
+  time.sleep(2)
```

Comment:
Comment why sleep?

Reply:
Good point, made this much cleaner and minimized the length of the sleep.